nrf_modem/doc/variant_features.rst

```diff
@@ -3,42 +3,66 @@
+  |                                 |                                +-------------+-------------+-----------------+-----------------+
+  |                                 |                                | mfw_nrf9160 | mfw_nrf91x1 | mfw_nrf9151-ntn | mfw-nr+_nrf91x1 |
+  +=================================+================================+=============+=============+=================+=================+
+  | Modem SoC management            | :file:`nrf_modem.h`            | Yes                                         | Yes             |
+  +---------------------------------+--------------------------------+---------------------------------------------+-----------------+
+  | :ref:`nrf_modem_fault`          | :file:`nrf_modem.h`            | Yes                                         | Yes             |
+  +---------------------------------+--------------------------------+---------------------------------------------+-----------------+
+  | :ref:`nrf_modem_trace`          | :file:`nrf_modem_trace.h`      | Yes                                         | Yes             |
+  +---------------------------------+--------------------------------+---------------------------------------------+-----------------+
+  | :ref:`nrf_modem_bootloader`     | :file:`nrf_modem_bootloader.h` | Yes                                         | Yes             |
+  +---------------------------------+--------------------------------+---------------------------------------------+-----------------+
+  | :ref:`nrf_modem_delta_dfu`      | :file:`nrf_modem_delta_dfu.h`  | Yes                                         | Yes             |
+  +---------------------------------+--------------------------------+---------------------------------------------+-----------------+
+  | :ref:`nrf_modem_log`            | :file:`nrf_modem_os.h`         | Yes                                         | Yes             |
+  +---------------------------------+--------------------------------+---------------------------------------------+-----------------+
+  | :ref:`nrf_modem_at` [#at-note]_ | :file:`nrf_modem_at.h`         | Yes                                         | Yes             |
+  +---------------------------------+--------------------------------+---------------------------------------------+-----------------+
+  | :ref:`nrf_modem_dect_phy`       | :file:`nrf_modem_dect_phy.h`   | No                                          | Yes             |
+  +---------------------------------+--------------------------------+---------------------------------------------+-----------------+
+  | :ref:`nrf_sockets`              | :file:`nrf_socket.h`           | Yes                                         | No              |
+  +---------------------------------+--------------------------------+---------------------------------------------+-----------------+
+  | :ref:`nrf_modem_gnss`           | :file:`nrf_modem_gnss.h`       | Yes                                         | No              |
```

Comment:
May be the entries in the cellular and DECT NR+ columns be center aligned to be more clear that for which firmware the values are applicable?